ztp/ran-crd/site-config-crd.yaml

```diff
@@ -77,23 +77,6 @@ spec:
                       baseDomain:
                         description: BaseDomain is the base domain to which the cluster should belong.
                         type: string
-                      clusterType:
-                        description: clusterType either standard or sno
-                        type: string
-                        default: standard
-                        enum:
-                          - standard
-                          - sno
-                      numMasters:
-                        description: Number of masters for the control plane.
-                        type: integer
-                        default: 1
-                        enum:
-                          - 1
-                          - 3
-                      numWorkers:
-                        description: Number of workers for the control plane.
-                        type: integer
                       clusterProfile:
```

Comment:
would u clean the cluster Profile as well, no use for it.

Reply:
Sure!